src/components/Header/components/SearchForm.tsx

```diff
@@ -0,0 +1,117 @@
+import { Box, chakra, useMultiStyleConfig } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import Button from "../../Button/Button";
+import ButtonGroup from "../../ButtonGroup/ButtonGroup";
+import Fieldset from "../../Fieldset/Fieldset";
+import Form, { FormField, FormRow } from "../../Form/Form";
+import Icon from "../../Icons/Icon";
+import Radio from "../../Radio/Radio";
+import RadioGroup from "../../RadioGroup/RadioGroup";
+import TextInput from "../../TextInput/TextInput";
+
+import { getEncoreCatalogUrl, getNYPLSearchURl } from "../headerUtils";
```

Comment:
Do you want the "R" in getNYPLSearchURl to be capitalized?

Reply:
Yeah, it should have the capital "L". I'll update.

Reply:
Funny.  I thought it was supposed to be `URI`.